eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -87,3 +88,4 @@ steps:
       -TeamReviewers "${{ parameters.GHTeamReviewersVariable }}"
       -Assignees "${{ parameters.GHAssignessVariable }}"
       -CloseAfterOpenForTesting $${{ coalesce(parameters.CloseAfterOpenForTesting, 'false') }}
+      -Draft $${{ coalesce(parameters.DraftPr, 'false') }}
```

Comment:
Following the lead here with `coalesce` though I'm not sure it's necessary.

Reply:
LOL I was going to ask the exact question I don't think it is necessary in either case.